### ISSUE TITLE
mzml format from 2020 Proteowizard output causes dinosaur to hang

### ISSUE THREAD

**Issue by chrashwood** [statisticalbiotechnology/quandenser#20]
Dear Matthew and Lukas,

I believe I've found a bug which depends on which version of Proteowizard is used to convert .raw files to .mzml.

I tested 4 versions of Proteowizard: 3.0.18267, 3.0.20003, 3.0.20018 and 3.0.20057. Only the 3.0.18267 version generated an mzml file that did not cause Dinosaur to hang. Using the verbose option in Dinosaur did not give any errors, just a hang.

Cheers,
Chris

**Comment by MatthewThe** [comment]
I'll see if I can do anything, but the problem is probably in one of the dinosaur dependencies. Would it be possible to share such an mzML file for testing?

**Comment by chrashwood** [comment]
I'm having trouble reproducing a file conversion that works (even with the correct Proteowizard version). As a result, I only have large files and can't prepare some smaller files. I'll spend some time later in the week trying to get it to work again.

**Comment by MatthewThe** [comment]
Apparently, the problem is that the new ProteoWizard versions add the line `<userParam name="ms level" value="1"/>` to the mzML file in the precursorList tag:

```
<precursorList count="1">
            <precursor spectrumRef="controllerType=0 controllerNumber=1 scan=50">
              <userParam name="ms level" value="1"/>
```

Removing these lines from the mzML file allows Dinosaur to process the file without issues. I'll try to make a pull request to Dinosaur's mzML reader repo to fix this.

**Comment by MatthewThe** [comment]
This should be temporarily solved by 2313a6b8da56d95e92d6802cdedec97db54ba375 awaiting the pull request https://github.com/fickludd/proteomicore/pull/1 (updated MzML reader for Dinosaur) for the permanent solution.
